spec3.yml

```diff
@@ -2497,6 +2501,10 @@ paths:
                     query:
                       type: string
                       example: jane
+                    emails[]:
+                      type : string
+                      description: Array of emails 
+                      example: jane.crandall@mail.com
```

Comment:
Wouldn't this need to be an array?

Reply:
Absolutely, I found the correct syntax this morning.